app/app/route.js

```diff
@@ -15,8 +15,12 @@ export default Ember.Route.extend({
     return tokens.join(' - ');
   },
 
+  setupController: function(controller, model){
+    controller.set('model', model);
+    controller.set('stack', model.get('stack.content'));
```

Comment:
IMO making `aptible-ability` async-aware would be the ideal. That sounds like a good direction.

But I think this is good to go if you simply add a `FIXME` on the previous line explaining that the `stack` is loaded by the `afterModel` hook, and that accessing `content` looks into the promise proxy for the real relationship model.